addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -1,6 +1,11 @@
 odoo.define('web_unsplash.image_widgets', function (require) {
 'use strict';
 
+const session = require('web.session');
+if (!session.is_admin) {
+    return;
+}
+
```

Comment:
Was there a reason to not let people use unsplash?

Reply:
The user needs to have unsplash rights to use it. Without this piece of code, the portal user gets an error when searching for an image. But this is not correct because a non admin user can have unsplash rights.
I will fix this

Reply:
That shouldn't be a blocking point, AFAIC.

Either the DB is used on our SaaS platform and then we use our own key instead so there is no limit (we have a enterprise license key).
Either the DB is self hosted and in this case we can assume:
1. The user enabled unsplash and created an account + generated a key. He knows what he is doing and if the rate limit is an issue for him with the default key (50req/h), he can just request a production key.
2. The user probably wants his registered user (portal users) to be able to have a better UX by allowing them to use unsplash on the forum/whatever.

I don't see any valid reason to alter that part of the media dialog for the portal user. What do you think?

If this is going too far for a bugfix and want to delegate that part to our team, don't hesitate to let me know.

Reply:
Also note that the image upload is/should be karma protected, so not everyone can do it.

@qsm-odoo Isn't really in favor of it. He doesn't really see the added value for a portal user to use unsplash image in the forum / etc.

That's still an open discussion then I guess, any opinion here @pierre-pde ? To summarize, we now allow (like a few years ago) portal user to access the media dialog, but prevent the use of Unsplash. I would allow them to use unsplash for all the reason I listed (and since I don't really feel like there is a reason not to). But indeed, the added value might not be there as @qsm-odoo points out.

Reply:
Okay I understand why it might be useful but I will wait for Pierre's opinion on this

Reply:
Yes, let's leave unsplash available if it's configured. 
If it's not configured, we have to make sure that portal user don't  see the unsplash configuration banner;
https://bit.ly/3jxDxcK